cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -57,24 +59,29 @@ function(handle_gyb_source_single dependency_out_var_name)
   endforeach()
   list(REMOVE_DUPLICATES gyb_expand_deps)
 
-  add_custom_command_target(
-      dependency_target
+  add_custom_command_target(dependency_target
       COMMAND
-          "${CMAKE_COMMAND}" -E make_directory "${dir}"
+        "${CMAKE_COMMAND}" -E make_directory "${dir}"
       COMMAND
-          "${PYTHON_EXECUTABLE}" "${gyb_tool}" "${gyb_flags}"
-          -o "${GYB_SINGLE_OUTPUT}.tmp" "${GYB_SINGLE_SOURCE}"
+        "${PYTHON_EXECUTABLE}" "${gyb_tool}" "${gyb_flags}" -o "${GYB_SINGLE_OUTPUT}.tmp" "${GYB_SINGLE_SOURCE}"
       COMMAND
-          "${CMAKE_COMMAND}" -E copy_if_different
-          "${GYB_SINGLE_OUTPUT}.tmp" "${GYB_SINGLE_OUTPUT}"
+        "${CMAKE_COMMAND}" -E copy_if_different "${GYB_SINGLE_OUTPUT}.tmp" "${GYB_SINGLE_OUTPUT}"
       COMMAND
-          "${CMAKE_COMMAND}" -E remove "${GYB_SINGLE_OUTPUT}.tmp"
-      OUTPUT "${GYB_SINGLE_OUTPUT}"
-      DEPENDS "${gyb_tool_source}" "${GYB_SINGLE_DEPENDS}" "${GYB_SINGLE_SOURCE}" "${gyb_expand_deps}"
-      COMMENT "Generating ${basename} from ${GYB_SINGLE_SOURCE} ${GYB_SINGLE_COMMENT}"
-      WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-      SOURCES "${GYB_SINGLE_SOURCE}"
-      IDEMPOTENT)
```

Comment:
I'd really rather not reformat this file in this patch, just to make it easier to verify subtle differences didn't alter the CMake semantics.

Reply:
I'll put it into a separate commit. Do you want me to make a new PR with it, or is keeping it in this one OK?

Reply:
It might(?) have broken the build as it is now? I think a new PR would be best, if you can.

Reply:
I'll do it in a new one 😄 
It looks like the CI failed with an intermittent error, but I'm not 100% sure. Once I remove the formatting lets try again?